util.js

```diff
@@ -15,25 +15,42 @@ module.exports = {
     copyWebResource: copyWebResource
 };
 
-function rewriteFile(args) {
+function rewriteFile(args, _this) {
     args.path = args.path || process.cwd();
     var fullPath = path.join(args.path, args.file);
 
-    args.haystack = fs.readFileSync(fullPath, 'utf8');
+    if (_this !== undefined) {
```

Comment:
no need to use if else and normal fs, you can directly do `args.haystack = _this.fs.read(fullPath);` as the module calling this will have fs available in the context, coz we add fs to the jhipsterFuc as well so its available in this always